crates/host/src/wasmbus/mod.rs

```diff
@@ -807,6 +911,9 @@ impl ActorInstance {
         operation: &str,
         msg: Vec<u8>,
     ) -> anyhow::Result<Result<Vec<u8>, String>> {
+        // Validate that the actor has the capability to call the target
+        ensure_actor_capability(self.claims.metadata.as_ref(), contract_id)?;
```

Comment:
This is the host receiving an invocation on behalf of an actor, right? This comment doesn't sound right

Reply:
Great catch!